### ISSUE TITLE
As a user, I want team reminders for PRs based on time since ready for review (vs time since opening)

### ISSUE THREAD

**Issue by abinoda** [pullreminders/backlog#24]
Currently the team reminder `threshold` settings sets how old a PR should be before it is included in reminders. The age is calculated by when the PR was opened.

It would be valuable for the threshold to be based on when a PR became ready for review so improve the signal vs noise level of team reminders.

It would be nice to have a few options (these should be moved under filters):

1. Time elapsed since a specific label was added (ie. `Ready for review`)
2. Time elapsed since first review was requested
3. Oldest review request
4. Time since PR was opened

**Comment by abinoda** [comment]
An example of a workflow that really relies on this from @traviskroberts:

> * once a PR review request is created, the user is DMed to notify them of the request
> * if they haven’t addressed the PR (either commented or approve/deny) within 45 minutes of the initial request, they would get sent a reminder

**Comment by abinoda** [comment]
````
Toggle review requests - using or not
  If so...  [ ] Review request threshold
              [ ] Remind authors

Move min age to filters..default to 0
```

Also see https://github.com/pullreminders/backlog/issues/37

**Comment by abinoda** [comment]
Thought...

If [Print and order by PR staleness instead of creation date](https://github.com/pullreminders/backlog/issues/57) negates the need for adding a staleness "filter", then maybe I can do something similar for review requests where I display the age of the (longest) review request as "time waiting in review", and order by that? And for "waiting on author", order by staleness?

This removes the need for configuration which is getting more and more complex.
